apps/frontend/src/renderer/components/roadmap/FeatureDetailPanel.tsx

```
@@ -161,14 +180,43 @@ export function FeatureDetailPanel({
             </h3>
             <div className="flex flex-wrap gap-1">
               {feature.dependencies.map((dep) => (
-                <Badge key={dep} variant="outline" className="text-xs">
-                  {dep}
-                </Badge>
+                <button
+                  key={dep}
+                  className="px-2 py-1 rounded-md text-xs font-medium bg-primary/10 text-primary hover:bg-primary/20 hover:underline cursor-pointer transition-all flex items-center justify-start gap-1"
+                  onClick={() => handleDependencyClick(dep)}
+                  title={`View dependency: ${getFeatureTitle(dep)}`}
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)